src/main/java/com/group6/harmoniq/models/User.java

```diff
@@ -63,6 +66,7 @@ public User(String spotifyId, String displayName, String email, int followers,
         this.externalSpotifyUrl = externalSpotifyUrl;
         this.topArtist = topArtist;
         this.topTrack = topTrack;
+        this.quizCount = 0;
```

Comment:
We already have a default value of 0 while declaring this property, here we can use a parameter from the constructor

Reply:
When changing the constructor the build checks were failing.

This commit passed though: 5ecf5932d49f343fe9c8e0494e7879f70d254823

Let me know if this is what you wanted!